code/datums/components/arable_turf.dm

```diff
@@ -0,0 +1,101 @@
+/datum/component/arable_turf
```

Comment:
at least a description of what this is would be nice

Reply:
Then I'm going to add a description... then rename because it can be used on more than just turfs...